### PR TITLE
Hw 13.1

### PR DESCRIPTION
**Pull request by gkhrshch**

### CODE REVIEW

File: src/main/java/mate/academy/internetshop/lib/Injector.java [hunk 1]
```diff
@@ -1,35 +1,99 @@
...
+        inject(ItemServiceImpl.class.getDeclaredFields());
+        inject(BucketServiceImpl.class.getDeclaredFields());
+        inject(Main.class.getDeclaredFields());
+    }
+
+    private static void inject(Field[] fields) throws IllegalAccessException {
         Class<UserDaoImpl> userDaoImplClass = UserDaoImpl.class;
-        Field[] consoleHandlerFields = consoleHandlerClass.getDeclaredFields();
-        for (Field field : consoleHandlerFields) {
+        Class<OrderDaoImpl> orderDaoImplClass = OrderDaoImpl.class;
```
Comment (Sofasmile) [inline src/main/java/mate/academy/internetshop/lib/Injector.java]: do we need this fields (from line 35 to 41)? try to remove them

File: src/main/java/mate/academy/internetshop/lib/Injector.java [hunk 2]
```diff
@@ -1,35 +1,99 @@
...
+        Class<ItemDaoImpl> itemDaoImplClass = ItemDaoImpl.class;
+        Class<BucketDaoImpl> bucketDaoImplClass = BucketDaoImpl.class;
+        Class<UserServiceImpl> userServiceImplClass = UserServiceImpl.class;
+        Class<OrderServiceImpl> orderServiceImplClass = OrderServiceImpl.class;
+        Class<ItemServiceImpl> itemServiceImplClass = ItemServiceImpl.class;
+        Class<BucketServiceImpl> bucketServiceImplClass = BucketServiceImpl.class;
+        for (Field field : fields) {
             if (field.getDeclaredAnnotation(Inject.class) != null
```
Comment (Sofasmile) [inline src/main/java/mate/academy/internetshop/lib/Injector.java]: we can make one if statement general:
if(field.getDeclaredAnnotation(Inject.class) != null){
if(...){...}
...
}

File: src/main/java/mate/academy/internetshop/service/BucketService.java [hunk 3]
```diff
@@ -1,8 +1,21 @@
...
+
+    Bucket update(Long id);
+
+    Bucket delete(Long id);
+
+    Bucket addItem(Long bucketId, Long itemId);
+
+    Bucket clear(Long bucketId);
+    
+    List getAllItems(Long bucketId);
```
Comment (Sofasmile) [inline src/main/java/mate/academy/internetshop/service/BucketService.java]:
```suggestion
    List<Item> getAllItems(Long bucketId);
```
the same in all methods

File: target/checkstyle-cachefile [hunk 4]
```diff
@@ -1,2 +1,30 @@
-#Thu Sep 12 16:48:52 EEST 2019
+#Mon Sep 16 09:52:17 EEST 2019
```
Comment (Sofasmile) [inline target/checkstyle-cachefile]: remove target folder from repository
use .gitignore file

File: src/main/java/mate/academy/internetshop/service/BucketService.java [hunk 5]
```diff
@@ -17,5 +18,5 @@
 
     Bucket clear(Long bucketId);
     
-    List getAllItems(Long bucketId);
+    List<Item> getAllItems(Long bucketId);
```
Comment (Sofasmile) [inline src/main/java/mate/academy/internetshop/service/BucketService.java:21]: do the same in all method which return list of elements

**Review by Sofasmile — APPROVED** [review]

File: src/main/java/mate/academy/internetshop/service/BucketService.java (hunk 5 above)
Comment (gkhrshch) [reply src/main/java/mate/academy/internetshop/service/BucketService.java:21]: found only one mistake of this kind

**Pull request merged**